src/main/java/com/gluonhq/substrate/target/AndroidTargetConfiguration.java

```diff
@@ -682,4 +685,32 @@ public BuildToolNotFoundException() {
             super("Android build tools not found. Please install it and try again.");
         }
     }
+
+    /**
+     * Scans the classpath and adds permission to AndroidManifest.xml for
+     * each attach service found.
+     * @param genManifest Path to the AndroidManifest.xml file
+     */
+    private void addRequiredPermissions(Path genManifest) {
+        try {
+            final List<String> attachServices = AttachResolver.attachServices(projectConfiguration.getClasspath());
+            attachServices.forEach(service -> {
+                try {
+                    switch (service) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Switch statements should have a default label](https://app.codacy.com/manual/gluonhq/substrate/pullRequest?prid=5533165)